src/AppBundle/Resources/javascript/RegistrationStateMachine.ts

```diff
@@ -0,0 +1,82 @@
+import { StatusPollService } from './StatusPollService';
+import { RegistrationStatusComponent } from './Component/RegistrationStatusComponent';
+import { Component } from './Component/Component';
+
+export class RegistrationStateMachine {
+  private IDLE = '1';
```

Comment:
Why are these status codes strings?

Reply:
The status codes are from tiqr library, maybe we should have converted the codes inside the Tiqr legacy adapter. Don't think it's worth the effort for now.